travel-planner/travel-planner-backend/app.js

```diff
@@ -1,10 +1,23 @@
 const morgan = require("morgan");
 const express = require("express");
 const cors = require("cors");
+require("dotenv").config();
```

Comment:
Nit: can we move it to a separate section to keep format looks better? 
Suggestion: move it to line 7 or 11 and add new empty lines to keep it separated from other imports.